core/templates/dev/head/components/RatingDisplayDirectiveSpec.js

```diff
@@ -21,7 +21,7 @@ describe('Rating display directive', function() {
 
   beforeEach(module('directiveTemplates'));
   beforeEach(module('oppia', GLOBALS.TRANSLATOR_PROVIDER_FOR_TESTS));
-  beforeEach(inject(function($rootScope, $compile, $templateCache) {
+  beforeEach(inject(function($compile, $rootScope, $templateCache) {
```

Comment:
^ I'm aware that it applies to `$scope` but can't recall if it applies to `$rootScope`

Reply:
If we put `$rootScope` or `$scope` at the start, before `$compile` or any other which lies above them in the alphabetical order, `angular/di-order` rule will give an error, stating that dependency injections are not arranged alphabetically. What should we do then?

Reply:
we may have to disable the check due to the custom order that we have, but let's see what Sean says.

Reply:
There's no specific condition, actually. Historically we've put $scope at the start since it applies to pretty much every controller, but I'm not really fussed either way as long as everything is consistent.

So, @apb7 -- feel free to make a decision, and ensure that everything is consistent with that decision!

Reply:
@seanlip: All controllers have been placed in alphabetical order now. The di-order rule will ensure that this consistency is maintained. So I think we can go with the alphabetical ordering of DIs. Thanks!